source/features/clean-repo-tabs.tsx

```diff
@@ -47,8 +47,16 @@ function onlyShowInDropdown(id: string): void {
 }
 
 const getWikiPageCount = cache.function(async (): Promise<number> => {
-	const wikiPages = await fetchDom(buildRepoURL('wiki'), '#wiki-pages-box .Counter');
```

Comment:
Unrelated but can you check if there’s an API for this? It looks like the search API has a wikiCount https://docs.github.com/en/graphql/reference/objects#searchresultitemconnection

That’s just about the only wiki reference in the whole GraphQL API (other than “has wiki enabled”)